mountpoint-s3/scripts/fs_bench.sh

```diff
@@ -30,6 +30,11 @@ if [[ -n "${S3_JOB_NAME_FILTER}" ]]; then
   echo "Will only run fio jobs which match $S3_JOB_NAME_FILTER"
 fi
 
+# Optional: Check if S3_ENDPOINT_URL is set; if not, use a default value
+if [[ -z "${S3_ENDPOINT_URL}" ]]; then
+  S3_ENDPOINT_URL="https://s3.amazonaws.com"  # Set your default endpoint URL here
+fi
```

Comment:
Instead of setting explicit default endpoint URL, I would suggest to add a new variable `optional_args` for optional arguments. It would be empty at first, then we can append anything we want and use this new variable in mount command.
```suggestion
optional_args=""

if [[ -n "${S3_ENDPOINT_URL}" ]]; then
  optional_args+="--endpoint_url=${S3_ENDPOINT_URL}"
fi
```